src/components/Meta.tsx

```diff
@@ -23,11 +23,19 @@ export default function render(
 
   let title:string;
   if(props.title){
-    title = props.title + " | SaaSquatch " + (props.categoryName || "Documentation");
+    title = props.title + " | SaaSquatch " + (props.categoryName || "Docs");
   }else{
-    title = "SaaSquatch Help Center";
+    title = "SaaSquatch Docs";
   }
-    
+  
+  let ogImageURL:string;
+  if(props?.fields?.ogFeaturedImage == null){
+    ogImageURL = null;
```

Comment:
Setting this to null feels a bit off, I think there is a cleaner way to achieve whats happening in this code block